source/Core/Threads/OperatingModes/ShowStartupWarnings.cpp

```diff
@@ -83,6 +84,23 @@ OperatingMode showWarnings(const ButtonState buttons, guiContext *cxt) {
 #else
     cxt->scratch_state.state1 = 4;
 #endif /*POW_PD_EXT==1*/
+#if POW_PD_EXT == 2
+    if (!FS2711::probe()) {
+      if (getSettingValue(SettingsOptions::PDMissingWarningCounter) < 2) {
+        if (warnUser(translatedString(Tr->NoPowerDeliveryMessage), buttons)) {
+          cxt->scratch_state.state1 = 4;
+          nextSettingValue(SettingsOptions::PDMissingWarningCounter);
+          saveSettings();
+        }
+      } else {
+        cxt->scratch_state.state1 = 4;
+      }
+    } else {
+      cxt->scratch_state.state1 = 4;
+    }
+#else
```

Comment:
This #else has been expanded incorrectly.
The main #if here needs to be moved into the else in the above state, so that when all the POW_PD defines are false we still only emit only one state1 change